synapse/lib/stormlib/gen.py

```diff
@@ -164,10 +164,29 @@ class LibGen(s_stormtypes.Lib):
                        'desc': 'Type normalization will fail silently instead of raising an exception.'},
                   ),
                   'returns': {'type': 'node', 'desc': 'An inet:tls:servercert node with the given server and SHA256.'}}},
+        {'name': 'platformByNameOrUrl',
```

Comment:
Should we consider breaking up the gen stuff into model areas? `$lib.gen.inet.platform()`?  ( I'd like to avoid the `ByFooOrBar()` if we think we can.